approved/j750/flow_control_flow.txt

```diff
@@ -217,11 +221,11 @@ DFF 1.1	Flow Table
 						logprint	This_should_generate_an_AND_flag																					
 						Test	test1	test1						None		at1_C2A232F_FAILED												
 						Test	test2	test2						None		at2_C2A232F_FAILED												
-						flag-true	at2_C2A232F_FAILED_AND_at1_C2A232F_FAILED																					
-						flag-false	at2_C2A232F_FAILED_AND_at1_C2A232F_FAILED															not	flag-true	at1_C2A232F_FAILED				
-						flag-false	at2_C2A232F_FAILED_AND_at1_C2A232F_FAILED															not	flag-true	at2_C2A232F_FAILED				
-						Test	test3	test3						Fail									flag-true	at2_C2A232F_FAILED_AND_at1_C2A232F_FAILED				
-						Test	test4	test4						Fail									flag-true	at2_C2A232F_FAILED_AND_at1_C2A232F_FAILED				
+						flag-true	at1_C2A232F_FAILED_AND_at2_C2A232F_FAILED																					
+						flag-false	at1_C2A232F_FAILED_AND_at2_C2A232F_FAILED															not	flag-true	at2_C2A232F_FAILED				
+						flag-false	at1_C2A232F_FAILED_AND_at2_C2A232F_FAILED															not	flag-true	at1_C2A232F_FAILED				
+						Test	test3	test3						Fail									flag-true	at1_C2A232F_FAILED_AND_at2_C2A232F_FAILED				
+						Test	test4	test4						Fail									flag-true	at1_C2A232F_FAILED_AND_at2_C2A232F_FAILED				

```

Comment:
Logically the same as the original, just the order of the names in the generated AND flag name is reversed, as is the order in which the flag is disabled based on one of the component flags being not set.